### ISSUE TITLE
Screen reader NVDA doesn't announce selected radio correctly

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#1641]
Description  
---
Browsing the `RadioWrapper` component with NVDA on Windows results in an unsufficient announcement of the selected radio element: "radio number **1 of 1** selected", but it should be something like "radio number **1 of 2** selected".

Notes
---
- Evaluate if Screen Reader works if radios are part of a form and wrapped in a fieldset: https://designsystem.porsche.com/latest/patterns/forms/example/various
- Moritz hat einen Windows Rechner ;)

To Reproduce
---
1. Go to on Windows with NVDA https://designsystem.porsche.com/v2/components/radio-button/examples 
2. Select radio element
3. See/hear error

Expected behavior
---
The number of the selected radio should be announced.

This works as expected:
```
<input type="radio" name="some-name-1" />
<input type="radio" name="some-name-1" />
```

This does not work as expected (missing number):
```
<p-radio-button-wrapper label="Some label">
  <input type="radio" name="some-name-1" />
</p-radio-button-wrapper>
<p-radio-button-wrapper label="Some label">
  <input type="radio" name="some-name-1" />
</p-radio-button-wrapper>
```

Test Environment
---
Platform/Browser/Version: [Windows, NVDA, Chrome]
